prov/gni/src/gnix_rma.c

```diff
@@ -125,19 +125,17 @@ static int __gnix_rma_send_completion(struct gnix_fid_ep *ep,
 	return FI_SUCCESS;
 }
 
-static void __gnix_rma_copy_indirect_get_data(struct gnix_tx_descriptor *txd)
+static void __gnix_rma_copy_indirect_get_data(struct gnix_fab_req *req)
```

Comment:
At this point, with a function this small, perhaps we could make this an inline function?